### ISSUE TITLE
Feedback feed skips H2 element

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1221]
Similar to #1181, the feedback feed skip the `<H2>` element (a violation of WCAG 2's Success Criteria 1.3.2):

```
<h1 class="devilry-group-feedbackfeed-header-assignment">
    Assignment 1
</h1>
...
<ol class="cradmin-legacy-listbuilder-list devilry-group-feedbackfeed-feed__feedbackset-wrapper" >
    <li>
    <div class="cradmin-legacy-listbuilder-itemvalue devilry-group-feedbackfeed-itemvalue devilry-group-feedbackfeed-feed__feedbackset-wrapper--header-first-attempt">
    <h3>
        <span>
            <span class="header-title">
                Deadline: Friday April 1, 2022, 23:59
            </span>
        </span>
    </h3>
```
